### PR TITLE
Fix Docker build

### LINKED ISSUE #67: Typescript not installed + other misc bugs
On building the latest PR:
```
We detected TypeScript in your project (src/index.tsx) and created a tsconfig.json file for you.

It looks like you're trying to use TypeScript but do not have typescript installed.
Please install typescript by running npm install typescript.
If you are not trying to use TypeScript, please remove the tsconfig.json file from your package root (and any TypeScript files).

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! webapp@0.1.0 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the webapp@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-08-27T15_56_49_803Z-debug.log
```

I managed to fix this on my deployment by doing this:
```
diff --git a/Dockerfile b/Dockerfile
index 6ee7fad..a14f84e 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -10,6 +10,7 @@ WORKDIR /app/webapp
 COPY webapp/package*.json ./

 RUN npm ci --only=production
+RUN npm install typescript

 # Copy source
 COPY webapp/src ./src
@@ -18,6 +19,7 @@ COPY webapp/public ./public
 # Build app
 ARG REACT_APP_ENDPOINT
 ENV REACT_APP_ENDPOINT=$REACT_APP_ENDPOINT
+ENV SKIP_PREFLIGHT_CHECK=true
 # For public resources
 ENV PUBLIC_URL=/static
```

Also seems like the theme is no longer saved to local storage

@codetheweb 

I'm unsure how to fix the theme saving at this moment.

### LINKED ISSUE #68: Add dockerbuild to github workflows
Code read my mind in DMs that we should have a docker build workflow to help catch bugs

### PR DESCRIPTION
**Pull request by codetheweb**

### CODE REVIEW

**Comment by codetheweb** [comment]
8f3df02 failed to build with the new Workflow as expected 👍

**Comment by codetheweb** [comment]
If @Blu3spirits can confirm that theme saving isn't broken, this should be good to go.

File: Dockerfile [hunk 1]
```diff
@@ -9,7 +9,7 @@ WORKDIR /app/webapp
 # Install packages
 COPY webapp/package*.json ./
 
-RUN npm ci --only=production
```
Comment (Blu3spirits) [inline Dockerfile]: I think we should just define typescript as a prod dep.

**Review by Blu3spirits — CHANGES_REQUESTED** [review]
> If @Blu3spirits can confirm that theme saving isn't broken, this should be good to go.

Wrote up my response here:
https://github.com/Kiro47/MTU-Transfer-Course-Gatherer/issues/67#issuecomment-682259743

TL;DR theme _saving_ wasn't broken, just theme loading due to a missing case of if storedTheme === dark

**Review by Blu3spirits — APPROVED** [review]
